app/views/users/index.html.erb

```diff
@@ -0,0 +1,8 @@
+  <div class="mb-auto d-flex align-items-center justify-content-center h-100">
+    <h1 class="text-center" >CountApp</h1>
+  </div>
+
+  <div class="d-flex flex-column mx-3">
+    <%= link_to "Log In", new_user_session_path, class: 'btn btn-primary p-3' %>
+    <%= link_to "sign up", new_user_registration_path, class: 'btn btn-light p-3' %>
+  </div>
```

Comment:
- [ ] As the previous reviewer mentioned kindly try to decrease the size of elements. I still cannot see some buttons on the app. 
 
Wireframe Layout   | Your Layout
------------- | -------------
<img width="268" alt="Screenshot 2022-05-27 at 17 11 26" src="https://user-images.githubusercontent.com/77038610/170705908-9a025cbb-452d-49f8-a1e3-63765c4f6bcf.png"> | <img width="392" alt="Screenshot 2022-05-27 at 17 07 24" src="https://user-images.githubusercontent.com/77038610/170705254-7b29218d-b9c0-4df9-9855-ba8b88fee496.png">

Wireframe button   | Your button
------------- | -------------
<img width="210" alt="Screenshot 2022-05-27 at 17 13 33" src="https://user-images.githubusercontent.com/77038610/170706266-6bc29974-5f1b-4424-8091-59a7d1a3f21e.png"> | <img width="396" alt="Screenshot 2022-05-27 at 17 12 54" src="https://user-images.githubusercontent.com/77038610/170706152-115e170e-4937-46c7-a79f-906a399c703a.png">

- [ ] Kindly make sure that the page fits the screen, the button on the bottom is always fixed on the bottom

- [ ] OPTIONAL: Based on requiements, kindly follow the these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), including:

Colors `#3778c2`, `#5fb523`, `#434b54`
Typography: font face, size and weight.
Layout: composition and space between elements.